pkg/components/velero/restic/restic.go

```diff
@@ -118,6 +118,18 @@ func (c *Configuration) Validate() hcl.Diagnostics {
 			Detail:   "Make sure to set region value is set",
 		})
 	}
+
+	for _, toleration := range c.Tolerations {
+		if toleration.TolerationSeconds != 0 && toleration.Effect != "NoExecute" {
```

Comment:
I think this should be a validation rule for `Toleration` type itself, so we do not duplicate it across components.